commercial/app/AppLoader.scala

```diff
@@ -56,8 +50,10 @@ trait CommercialServices {
   lazy val feedsParser = wire[FeedsParser]
 }
 
-trait AppLifecycleComponents {
-  self: FrontendComponents with Controllers with CommercialServices =>
+trait AppComponents extends FrontendComponents with CommercialControllers with CommercialServices {
+
+  lazy val devAssetsController = wire[DevAssetsController]
+  lazy val healthCheck = wire[HealthCheck]
```

Comment:
You're right, I think it looks cleaner and is probably easier to read like that!

I got `trait` crazy on this one.